presentation-compiler/src/main/dotty/tools/pc/AutoImports.scala

```diff
@@ -246,7 +246,7 @@ object AutoImports:
         // see WorksheetProvider.worksheetScala3AdjustmentsForPC
         val indent =
           if pos.source.path.isWorksheet &&
-            editPos.getStart().getCharacter() == 0
+            editPos.getStart.nn.getCharacter == 0
```

Comment:
```suggestion
            editPos.getStart().nn.getCharacter() == 0
```
I guess this errors out only for Scala code, but I would keep it consistent.